sdk/storage/storage-file-share/review/storage-file-share.api.md

```diff
@@ -710,6 +745,44 @@ export { HttpRequestBody }
 
 export { IHttpClient }
 
+// @public
+export interface Lease {
```

Comment:
Sounds not accurate for the `Lease` interface naming. It more likes a "LeaseOperationResponseHeaders" instead of "Lease".

Reply:
Indeed. Fixed.